corehq/apps/data_interfaces/tests/test_utils.py

```diff
@@ -542,14 +542,10 @@ def create_repeat_records(cls):
             )
             couch_record.save()
             cls.couch_records.append(couch_record)
-
-            cls.sql_records.append(SQLRepeatRecord.objects.create(
-                domain=DOMAIN,
-                couch_id=couch_record._id,
-                payload_id=cls.instance_id,
-                repeater=cls.repeater,
-                registered_at=now,
-            ))
+        cls.sql_records = list(SQLRepeatRecord.objects.filter(
```

Comment:
nit: You can get rid of line 534 (cls.sql_records = [])

Reply:
Thanks. This gets refactored in a [later commit](https://github.com/dimagi/commcare-hq/commit/3cd0bece1c890fd210b2ea2bc4f822aac5db3cce) (it will be in the next PR).